gravitee-am-ui/src/app/domain/settings/users/creation/user-creation.component.ts

```diff
@@ -73,6 +74,9 @@ export class UserCreationComponent implements OnInit {
       this.userService.isEmailRequired().subscribe((response: boolean) => {
         this.emailRequired = response;
       });
+    } else {
+      console.log('pass policy', this.passwordPolicy);
+      console.log('profile', { id: this.user.id, firstName: this.user.firstName, lastName: this.user.lastName, email: this.user.email });
```

Comment:
question: do we want to keep it ? If so why log those info only for org user?

Reply:
after another look, these logs don't give any valuable info after I've fixed the issue, i've removed all of them again